submissions/Valkirin/frontend-nanodegree-arcade-game/js/app.js

```diff
@@ -0,0 +1,80 @@
+var Enemy = function(x, y, speed) {
+    this.x = x;
+	this.y = y;
+	this.speed = speed;
+    this.sprite = 'images/enemy-bug.png';
+};
+
+Enemy.prototype.update = function(dt) {
+    this.x +=  + (this.speed * dt);
+    
+    if (this.x > 550) {
+    	this.x = -100;
```

Comment:
-100 is a mystery number. We write code for other people in the first hand.
Use constants. Consider grouping constants into object literals.
Fix here and elsewhere in the code.

Reply:
fixed